File: hw/bonilla/U3/HW18AbstractFactory/GUIFactoryJs/model/WinFactory.js
const GUIFactory = require('../controller/GUIFactory');
const WinButton = require('../view/WinButton');
const WinMenu = require('../view/WinMenu');

class WinFactory extends GUIFactory {
  createButton() {
    return new WinButton();
  }

  createMenu() {
    return new WinMenu();
  }
}

module.exports = WinFactory;

File: hw/sanchez/u3/HW18GUIFactory/GUIFactory/src/main/java/ec/edu/espe/guifactory/model/WinFactory.java
package ec.edu.espe.guifactory.model;

import ec.edu.espe.guifactory.controller.GUIFactory;
import ec.edu.espe.guifactory.view.WinButton;
import ec.edu.espe.guifactory.view.WinMenu;

/**
 *
 * @author Jahir Sivinta DDCO
 */
public class WinFactory extends GUIFactory { 
    public Button createButton() { 
        return(new WinButton());
    }
    public Menu createMenu() {
        return(new WinMenu());
    }
}
